src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Homework19 extends BaseTest {
+    @Test(dataProvider = "invalidCredentials", dataProviderClass = BaseTest.class)
+    public void deletePlaylist() {
+        provideEmail("denise626@hotmail.com");
+        providePassword();
+        clickSubmitBtn();
+
+        WebElement choosePlaylist = driver.findElement(By.cssSelector("[href = '#!/playlist/17356']"));
+        choosePlaylist.click();
+        WebElement deleteSongPlaylist = driver.findElement(By.cssSelector("[id='extraTabAlbum']"));
+        deleteSongPlaylist.click();
+
```

Comment:
Then you need to click on the "OK" button

Locator  `button.ok`

![image](https://user-images.githubusercontent.com/104761482/198518996-aaf229af-19e6-4d35-a224-ae3a0ebdd38a.png)